app.js

```diff
@@ -20,6 +20,7 @@ initPassport();
 
 app.use(cors());
 
+
```

Comment:
Please revert this change since you didn't do anything in this file.

Reply:
Ok. Noted